apps/sms/index.html

```diff
@@ -84,6 +84,9 @@
     <script defer src="views/shared/js/settings.js"></script>
     <script defer src="services/js/message_manager.js"></script>
     <script defer src="views/shared/js/app.js"></script>
+    <script defer src="services/activity/threads.js"></script>
```

Comment:
unless everyone agrees with the name "bridge". (I think Wilson didn't like it but I don't remember why)